spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -372,16 +375,16 @@ will emit a message with a TaskLaunchRequest object containing the required laun
 would be to launch the timestamp task once every 5 seconds, the stream to implement this would look like:
 
 ```
-stream create foo --definition "triggertask --triggertask.uri=maven://org.springframework.cloud.task.app:timestamp-task:jar:1.0.0.BUILD-SNAPSHOT --trigger.fixed-delay=5 | task-launcher-local" --deploy
+stream create foo --definition "triggertask --triggertask.uri=<uriToJar> --trigger.fixed-delay=5 | task-launcher-local" --deploy
 ```
 
-=== Translator
+=== TaskLaunchRequest-transform
 
-Another option to start a task using the `task-launcher` would be to create a stream using a your own translator
-(as a processor) to translate a message payload to a TaskLaunchRequest.  For example:
+Another option to start a task using the `task-launcher` would be to create a stream using the
+https://github.com/spring-cloud/spring-cloud-stream-app-starters/tree/master/processor/spring-cloud-starter-stream-processor-tasklaunchrequest-transform[Tasklaunchrequest-transform] processor to translate a message payload to a TaskLaunchRequest.  For example:
```

Comment:
We can change it on the app side.  But that would be after this release I'd imagine.